CHANGELOG.md

```diff
@@ -18,6 +18,7 @@ git logs & PR history.
 ### Added
 
 - `--preserve-state` flag will preserve a given host options hash across subcommand runs(BKR-1541)
+- `BEAKER_HYPERVISOR` environment variable to choose the beaker-hostgenerator hypervisor
```

Comment:
@ekohl please move this changelog entry into the "Unreleased" section above

Reply:
Updated